src/modules/item/export.ts

```diff
@@ -15,9 +17,17 @@ export async function deployContents(identity: AuthIdentity, collection: Collect
   const [files, image] = await Promise.all([getFiles(item.contents), generateImage(item)])
   const contentFiles = await makeContentFiles({ ...files, [IMAGE_PATH]: image })
   const catalystItem = toCatalystItem(collection, item, chainId)
-  const [data] = await buildDeployData(EntityType.WEARABLE, identity, [urn], catalystItem, contentFiles, ITEM_DEPLOYMENT_DELTA_TIMESTAMP)
+  const client = new CatalystClient(PEER_URL, 'Builder')
+  const { entityId, files: hashedFiles } = await client.buildEntity({
+    type: EntityType.WEARABLE,
+    pointers: [urn],
+    metadata: catalystItem,
+    files: contentFiles,
+    timestamp: Date.now() - ITEM_DEPLOYMENT_DELTA_TIMESTAMP * 1000
```

Comment:
@nachomazzara I multiplied delta by 1000, because timestamp is in millis, not in seconds